lua/aerial/backends/treesitter/init.lua

```diff
@@ -48,8 +49,15 @@ M.fetch_symbols_sync = function(bufnr)
   -- It is used to determine node parents for the tree structure.
   local stack = {}
   local ext = extensions[lang]
-  ---@diagnostic disable-next-line: missing-parameter
-  for _, matches, metadata in query:iter_matches(syntax_tree:root(), bufnr) do
+  for _, matches, metadata in
```

Comment:
I am not really sure what should be passed in for 3th and 4th parameters so I just followed the neovim documentation.